salt/engines/slack.py

```diff
@@ -224,5 +224,6 @@ def fire(tag, msg):
                     else:
                         # Fire event to event bus
                         fire('{0}/{1}'.format(tag, _m['type']), _m)
+            time.sleep(1)
```

Comment:
This indentation level doesn't line up. Could you adjust it to align with the instructions in the block that you want to be executed? Thanks.